src/js/src/components/SearchBox.vue

```diff
@@ -14,23 +14,31 @@
               id="clearSubmit"
               title="Clear search and results"
               type="button"
-              @click="clearResultsAndSearch" />     
+              @click="clearResultsAndSearch" />
     </form>
+    <div @click="showUploadFileSearch = !showUploadFileSearch">
+      <span class="searchBoxActionLink">Search with uploaded file</span>
+    </div>
     <applied-search-facets />
+    <search-upload-file v-if="showUploadFileSearch" />
```

Comment:
The placement of the button is a little weird if you start a search and place some filters. Perhaps we could find a better way to start the interaction - maybe clear the search all together and push a new state when you want to search for an uploaded file?

Reply:
I totally agree here and actually I forgot about testing with filters applied :) I think you are spot on with the interaction - we should consider removing or making the search box  (appear) inactive when searching with uploaded file.